pandas/tests/indexes/test_multi.py

```diff
@@ -78,6 +78,31 @@ def test_labels_dtypes(self):
         self.assertTrue((i.labels[0] >= 0).all())
         self.assertTrue((i.labels[1] >= 0).all())
 
+    def test_repeat(self):
```

Comment:
No need.  I'm just overriding the original test with this one specific to `MultiIndex`.